pages/blog/caching-data-with-dataloader.mdx

```diff
@@ -0,0 +1,137 @@
+import withArticle from '../../ui/blog/article'
+export const meta = {
+  title: 'Caching data with DataLoader',
+  author: 'giladtidhar',
+  tags: ['cache', 'data', 'servers', 'graphql'],
+  date: '2022-1-12',
+  description: 'How to use DataLoader to cache data',
+  image: '/blog-assets/caching-data-with-dataloader/cover.png'
+}
+export default withArticle({ ...meta })
+
+# Caching
+
+Caching data is an important thing, it could be the difference between a slow wasteful server and a fast efficient server.
+
+But, what is caching?
+A cache is a place in memory for important data, which can be delivered to the client faster than the regular procedure.  
```

Comment:
```suggestion
Caching is the technical design of trying to avoid compute-intensive tasks by keeping temporarily their results in a cache.
A cache is a place in memory for important data, which can be delivered to the client faster than the regular procedure.

For example, for a GraphQL API, we might want to cache a field value that comes from a slow external API.
```